airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -223,6 +221,13 @@ class KubernetesPodOperator(BaseOperator):
         container name to use.
     :param deferrable: Run operator in the deferrable mode.
     :param poll_interval: Polling period in seconds to check for the status. Used only in deferrable mode.
+    :param on_finish_action: What to do when the pod reaches its final state, or the execution is interrupted.
+        If "delete_pod", the pod will be deleted regardless it's state; if "delete_succeeded_pod",
+        only succeeded pod will be deleted. You can set to "keep_pod" to keep the pod.
```

Comment:
How do we feel about
`cleanup_action`?
with options `keep`, `delete`, `delete_succeeded` ?

Reply:
`on_finish_action` is better IMHO -> `cleanup = keep` sounds redundant and contradicting (are we cleaning up really if we keep it ?)

Reply:
I think the current action names are more clear. However, for the argument name, I have no preference between `cleanup_action` and `on_finish_action`; both are meaningful and suitable.

Reply:
Cool lets stay with that